runtime/lua/vim/fs.lua

```diff
@@ -208,19 +211,19 @@ end
 
 --- Normalize a path to a standard format. A tilde (~) character at the
 --- beginning of the path is expanded to the user's home directory and any
---- backslash (\\) characters are converted to forward slashes (/). Environment
+--- backslash (\) characters are converted to forward slashes (/). Environment
```

Comment:
This disappears in the generated docs.

Reply:
@AzerAfram don't add it back manually to the documentation. Revert this change, this is wrong.

Reply:
What do you mean?

Reply:
That having `(\)` is wrong here. It should be `(\\)` like it was before.

Reply:
The actual documentation is generated based on this. We are not updating two separates files manually with the same change. We have the `./scripts/gen_vimdoc.py` script to automate that.

Reply:
I don't know why it has two backslashes there it's supposed one, look at `neovim/runtime/doc/lua.txt` line 2312.

Reply:
This is not `lua.txt`, this is `fs.lua`. `lua.txt` is generated based on this file (and some other files too).

Two backlashes here wasn't a mistake, the first backslash escapes the other. One backslash here won't generate anything in `lua.txt`, and two backslashes will translate to one backslash.